file_paths.md

```diff
@@ -1,24 +1,325 @@
 Puppet Installation Layout
 ==========================
 
-This table specifies the file paths in a Puppet installation and the corresponding settings, denoted as `:setting`. The 3.x column shows the difference of the specification vs the 3.x implementation.
+These tables specify the file paths in a Puppet installation and the corresponding settings, denoted as `:setting`.
 
 # Index
 
-* [puppet-agent (*nix)](#puppet-agent-nix)
-* [puppet-agent (Windows)](#puppet-agent-windows)
+ Index
+
+* [puppet-agent 6 (*nix)](#puppet-agent-6-nix)
+* [puppet-agent 6 (Windows)](#puppet-agent-6-windows)
+* [puppet-agent 4, 5 (*nix)](#puppet-agent-4-5-nix)
+* [puppet-agent 4, 5 (Windows)](#puppet-agent-4-5-windows)
 * [puppet-agent (non-root)](#puppet-agent-non-root)
 * [puppetdb](#puppetdb)
 * [puppetserver](#puppetserver)
 * [puppetmaster](#puppetmaster)
 * [razor-server](#razor-server)
 * [Notes](#notes)
 
-# puppet-agent (*nix)
+# puppet-agent 6 (*nix)
+
+The package will create the following services, all running as `root`
```

Comment:
Is there a tl;dr of what changed from Puppet 5?

Reply:
Sure, I've updated this to add some change summaries to the preambles before the Puppet 6 sections